components/ui/Button.tsx

```diff
@@ -1,12 +1,12 @@
-import type { ButtonProps } from '#/shared/ui/props/buttonProps.ts';
+import { ButtonProps } from "#/components/ui/props/buttonProps";
 
-export function Button({ onClick }: ButtonProps) {
+export function Button({ onClick, children }: ButtonProps) {
   return (
     <button
       className="bg-primary-500 hover:bg-primary-400 hover:border-primary-400 text-black"
       onClick={onClick}
     >
```

Comment:
You can provide all standart button attributes as shown below, so you can use it like `<button>` tag (e.g. set `disabled`) and don't need to add attributes one by one if you need something later.

```suggestion
export function Button({ onClick, children, ...props }: PropsWithChildren<ButtonHTMLAttributes<HTMLButtonElement>>) {
  return (
    <button
      className="bg-primary-500 hover:bg-primary-400 hover:border-primary-400 text-black"
      onClick={onClick}
      {...rest}
    >
    {children}
```